content/docs/core-principles/beehaw-is-a-community.md

```diff
@@ -5,23 +5,23 @@ weight: 20
+From the early stages of conceptualization of what we wanted to do differently, up through the feedback we’ve been getting as Beehaw has been growing, there’s been a consistent narrative and push back from certain individuals about how we’ve decided to run things here. To be clear, these are the individuals whom are either on the fence, those who are not enthusiastic about our mission and voice it elsewhere, and to a lesser extent some of the individuals we have since banned from our platform. The narrative typically takes the position that ‘open/free speech’ is paramount and that any suppression of said speech is unwelcome (typically said in a much more hostile way). As we’ve experienced this push back, we’ve slowly gathered our thoughts and realized what we believe is a fundamental disconnect between those who have earnestly and openly adopted our platform and those who fight against it.
 
 ## Beehaw is a community. Communities are organic. 
 As a community grows and shrinks, everything about the community fundamentally changes. Most online social spaces don’t operate as communities on the same level that communities do offline. When communities are run in a way that the members of the community do not like, the community often splinters, or leaders are ousted. Websites tend to have much stronger incentives to stay on a platform and leaders (platforms) are much more resistant to this kind of natural control by the members of the platform - you can’t exactly overthrow Facebook. However, communities still need to have some kind of rules, and because the size of a community is much more amorphous online - but in general much larger - the default state we’re used to online is one of semi-authoritarianism with explicit rules.
 
-If you’ve ever spent some time deeply involved in an offline community, especially if you’ve done so as an organizer or otherwise involved in the management or running of a community, you’re probably at least somewhat aware of the kinds of discussions that communities regularly need, in order to keep them running. Communities are not perfectly homogeneous, and many communities value diversity. However, get enough humans together and there will always be disconnects of values, boundaries, wants, and needs. Navigating these disconnects **can be as simple as ensuring that two people don’t sit near each other at an event or as difficult as engaging the majority of the community in a discussion about what kinds of behavior are acceptable and what aren’t**. Discussions happen at all kinds of different levels and involve different groups of people to reflect where the disconnect happened and involve the parties necessary to resolve the disconnect as well as to manage the emotions, needs, wants, values, or boundaries of people who were hurt when this disconnect happened.
```

Comment:
I can't tell what changed here on or on life 17.

Reply:
Way easier to see in the rich diff, though then you lose line numbers

https://github.com/beehaw-org/beehaw-org.github.io/pull/8/commits/f7e9965bbced3fc939598ba1b456a97ebdf2bb7c?short_path=ecddd18#diff-ecddd182da16c9a5e1457f9d087d482bbd0984a318bf151a46206a7a613c3ac0

I think for future PRs I'll squash everything down to one commit to make it easier to flip into that mode, with the exception of actual content additions like cb50042 . Thanks for patience as I mess around with this.